modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda it’s worth ensuring that the cluster is deployed and configured according to Redpanda Data recommendations. The checklist in this guide will help you ensure that you are set up for success. In this guide we identify three levels of production readiness - level 1 through level 3. At level 3 you will be ready to run mission critical workloads in the largest of enterprises, whereas smaller organizations or less critical workloads may suffice at level 1.
+
+Redpanda provides multiple resources for deploying a production cluster. We recommend you utilize the Automated deployment instructions with the Redpanda Ansible collection. If you cannot use this method, we provide manual deployment instructions, which are more involved.
+
+== Checklists
```

Comment:
Given that it is just a list of links, we don't need the "Checklists" heading here. Each list can appear in the intro under "Level 1 checklist", "Level 2 checklist", and so on. Users can still easily click back to the checklist of choice in the right nav.